axis_linear: 4
scale_linear: 0.2
scale_linear_turbo: 0.1

axis_angular: 3
scale_angular: 0.2

axis_arm_yaw_head: 0
scale_arm_yaw_head: 0.3

axis_arm_pitch_head: 1
scale_arm_pitch_head: 0.01

axis_arm_endpoint_up_down: 7
scale_endpoint_up_down: 0.1

axis_arm_endpoint_forward_backward: 6
scale_endpoint_forward_backward: 0.1

button_flipper_front_up: 3
button_flipper_front_dn: 0
scale_flipper_front: 0.3

button_rotate_arm_left: 2
button_rotate_arm_right: 1
scale_rotate_arm: 0.2

enable_button: 5  # R1 shoulder button
enable_turbo_button: 4  # L1 shoulder button
